external-crates/move/crates/move-vm-runtime/src/runtime/data_cache.rs

```diff
@@ -123,6 +123,7 @@ impl<S: ModuleResolver> DataStore for TransactionDataCache<S> {
                         account_cache.module_map.clone(),
                         account_cache.original_id,
                         *package_id,
+                        0,
```

Comment:
This is the exact sort of thing I was afraid of. This instance is okay since this is getting removed in #23729